### PR TITLE
Fix reset password url bug

### PR DESCRIPTION
**Pull request by patjouk**
Link was not working ;)

### CODE REVIEW

File: core/forms.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ class UserChangeForm(forms.ModelForm):
         label="Password",
         help_text="Raw passwords are not stored, so there is no way to see "
                   "this user's password, but you can change the password "
-                  "using <a href=\"password/\">this form</a>.")
+                  "using <a href=\"../password/\">this form</a>.")
```
Comment (aniav) [inline core/forms.py]: Could we use a reverse url here? We should avoid using hardcoded urls, because they might break without our notice :)

Comment (patjouk) [inline core/forms.py]: I tried again and couldn't find how to do it. We need the user ID to generate the URL but I don't know how to get it here.
Do you want to try?

Comment (aniav) [inline core/forms.py]: Checking :)

Comment (aniav) [inline core/forms.py]: @patjouk it seems we were using of Django's Form copied to our source code. I have removed our copy and now the link should be fine :)

**Pull request merged**